src/room/GroupCallView.tsx

```diff
@@ -249,11 +249,13 @@ export const GroupCallView: FC<Props> = ({
     }
   }, [isJoined, rtcSession]);
 
-  const e2eeConfig = useMemo((): E2EEConfig | undefined => {
+  const e2eeConfig = useMemo((): E2EEConfig => {
     if (perParticipantE2EE) {
       return { mode: E2eeType.PER_PARTICIPANT };
     } else if (e2eeSharedKey) {
       return { mode: E2eeType.SHARED_KEY, sharedKey: e2eeSharedKey };
+    } else {
+      return { mode: E2eeType.NONE };
```

Comment:
we also have the e2eeEnabled prop. maybe a check if `e2eeEnabled=false` should overrride this?

Reply:
ah, so this is a throwback from full mesh and passes the flag to the js-sdk to disable encryption of the events for calling, so not relevant for livekit. In fact, we should probably remove it altogether from the livekit branch as it won't do anything. This is probably not something for this PR though as it's not particularly dependant or related.